### ISSUE TITLE
Simplify the DividerLine component

### ISSUE THREAD

**Issue by ivarni** [SpareBank1/designsystem#618]
#### Which package is this issue related to?

`ffe-core-react`, `ffe-core`

#### Describe your issue (screenshots welcome!)

The `DividerLine` component doesn't really make much sense in its current implementation. It will render an empty `<hr>` element and then 
a) Mark it with `aria-hidden="true"`
b) Mark it with `role="presentation"`

Looking at the [spec](https://www.w3.org/TR/wai-aria-practices/#presentation_role) there's a couple of reasons why this does not make sense.

For starters, the presentation role indicates *"that an element is being used only for presentation and therefore does not have any accessibility semantics"* so we're basically just grabbbing a semantic element and then stripping it of its semantics.

Second, the presentation role is ignored by browsers if *"[t]he element has any of the twenty-one global ARIA states and properties"*, so using `aria-hidden` on the semantic element we wanted to strip of its semantics negates that role.

Third, why are we using `aria-hidden` on an empty element in the first place?

#### What is the expected behavior? 

This should just be an empty `div` with the `ffe-divider-line` class by default and allow the consumer to override that default element with a prop if needed.

**Comment by ivarni** [comment]
Also note that the browser default styling of the `<hr>` element is not reset by the the `ffe-core` CSS. It seems to be `margin: 0.5em 0` across browsers but it may be diffferent in different browsers and should be set explicitly.

**Comment by ivarni** [comment]
@42tte While we're doing a11y stuffs anyway, do you have any thoughts on this?

**Comment by steffenz** [comment]
Hi!
I tried to look into this issue, but I'm not sure if I fully understand all the details of it. I've always though of HR as more of a visual break in the page flow, but I see now that this might not always be the case. I found some tips in the [WAI Accessibility Guidelines](https://www.w3.org/WAI/GL/techniques.htm), for example:

> For graphics used as section separators (e.g., horizontal rules) provide information about what the graphic represents.  In other words, provide an equivalent textual cue as to what the visual reader perceives (Example 1).  However, we recommend using the HR element with style sheets and the "title" attribute (Example 2). 

I'm not sure if changing this to a DIV would solve the actual problem of it being used the wrong way. Again - I might be missing/not understanding something here. Maybe we should enforce a title instead? 

Do you have any other suggestions, either here or as code that I can look at?

**Comment by ivarni** [comment]
I don't really know what the correct approach here is, I just thought we were doing it the wrong way. We're still going out of our way to use a semantic element and then at the same time trying to remove its semantics. 

I can't recall any longer why I landed on the suggested fix, which is why I reached out for other opinions. As far as I know we've had a few accessibility audits without this being brought up as an issue so maybe it really isn't.
